core/src/main/resources/hudson/diagnosis/ReverseProxySetupMonitor/resources.js

```diff
@@ -0,0 +1,35 @@
+(() => {
```

Comment:
Arrow functions do not work on some browsers jenkins supports. Basically, adjuncts cannot use them or other ES6 features.